gen/installer/azure.py

```diff
@@ -20,7 +22,7 @@
 
 TEMPLATE_PATTERN = re.compile('(?P<pre>.*?)\[\[\[(?P<inject>.*?)\]\]\]')
 
-UPLOAD_URL = ("https://az837203.vo.msecnd.net/dcos/{channel_commit_path}/azure/{arm_template_name}")
```

Comment:
This is the main fix... we still had some remnants of the Azure storage location hard-coded.